coss/opensource_clubs/models.py

```diff
@@ -41,6 +41,24 @@ def get_first_image(self):
     ]
 
 
+class AboutPage(Page):
```

Comment:
Regarding this page, I believe we need a more modular design. We want a generic design/template where we can use this template in multiple sites, at least the ones that will have an `about` page. 

I would suggest to add an as much as possible generic design in the `home` app. From there you can override it here and add the fields that are unique to the OSCs.

Reply:
This also means that the CSS/JS libs need a restructure to follow the modular design.

Reply:
Instead of adding a default image, you need to give the ability to content creators to upload any image.